charts/postgresql/values.yaml

```diff
@@ -1,6 +1,8 @@
 # Configuration for postgres subchart
 enabled: true
 
+verify_ssl: false
```

Comment:
@andriisoldatenko nit but could we make it `verifySSL` for consistency with the rest of the umbrella?